src/app/components/Navbar/Navbar.tsx

```diff
@@ -39,6 +40,7 @@ export default function Navbar({ title, subtitle, children }: Props) {
         </div>
       )}
       <div className="md:w-4/12 lg:w-3/12 flex justify-end items-center">
+        <LocaleSwitcher />
```

Comment:
Not sure if we need this in the navbar or if it would be ok to have this i.e. under settings

Reply:
this will be a case of dropdown under dropdown! mostly locale switchers shall be handy so I thought navbar shall be the best place! 
keeping it in settings and then it might happen that user never knows that we support multiple languages

Reply:
I prefer on settings too.

Reply:
Sure, I understand. Just thinking about this regarding things being stuffed in the navbar and if this is really needed or not. If the user is already using the browser in a certain language and alby is using that as a default then chances are high the user doesn't need to adjust the language all the time. Just wanted to give this a thought.  
@bumi @MoritzKa what do you think?

Reply:
was thinking the same. I guess the user does not change the language often. I'd prefer to save the user's language in the settings. If we can default to the browser's language then this should be fine, isn't it?

Reply:
okay so moving locale switcher under settings

Reply:
I moved the locale switcher under settings! shall we preserve the locale along with other settings?

Reply:
the problem I am getting is I am not able to do ```saveSettings``` from the intro page. where locale switcher is present as well.

Reply:
as we are using a browser language detector! so this will handle the case for new users! 

for existing users locale is persisted by default which is changed from language switcher, so do we really need to store locale in local browser storage?

Reply:
I think we can go with not storing it for now and adjust later if needed. Shouldn't block this.